lib/prawn/icon/interface.rb

```diff
@@ -110,7 +110,7 @@ def inline_icon(text, opts = {})
         box_options = opts.merge(
           inline_format: true,
           document: self,
-          at: [bounds.left, cursor]
+          at: [opts[:x].present? ? opts[:x] : bounds.left , opts[:y].present? ? opts[:y] : cursor]
```

Comment:
We should be able to use Ruby's [`Hash#fetch`](https://ruby-doc.org/core-2.5.1/Hash.html#method-i-fetch)  method with a default block value.

What do you think of this?

```suggestion
          at: [
            opts.fetch(:x) { bounds.left },
            opts.fetch(:y) { cursor }
          ]
```

This should fix up the test suite as well.